git/foundations_git/commit_messages.md

```diff
@@ -20,14 +20,43 @@ Yes! Let me give you a quick list of reasons why:
 
 -   Having a good commit message history will also be helpful if you come back to a project you were working on after stepping away from it for a while. You likely won't remember your thought process and changes made when initially writing out the code. 
 
+### Bad vs. Good Commits
+
+When it comes to writing commits, it is crucial to know how to write them effectively. Here's an example of a bad commit message:
+
+`fix a bug`
+
+Even though it describes what you did, the message is too vague, which leaves the other developers on your team confused. A good commit message will have a body and a subject. Here is a template to keep in mind when creating your commit message.
+
+## Subject
+An abbreviation that specifies an action that your line of code is doing(e.g., feat and fix) 
+
+`Commit type:`
+
+## Body
+A concise yet clear description of what you did 
+
+`Commit type: body`
```

Comment:
I think theres a bit of a misunderstanding @CBID2, this isn't a body, it's a subject.

If you haven't already, I recommend giving [the assignment](https://cbea.ms/git-commit/) in this lesson a quick read. It explains commit subject and body (with examples) in more detail and should give you a better idea.